support/COMMUNITY_CONTACTS.md

```diff
@@ -62,30 +62,32 @@ your timeslot with others by sending PRs.
 - [lionelvillard](https://github.com/lionelvillard)
 - [matzew](https://github.com/matzew)
 - [n3wscott](https://github.com/n3wscott)
-- [nachocano](https://github.com/nachocano)
 - [slinkydeveloper](https://github.com/slinkydeveloper)
 - [vaikas](https://github.com/vaikas)
 - [lberk](https://github.com/lberk)
 - [pierDipi](https://github.com/pierDipi)
 - [aliok](https://github.com/aliok)
+- [zhongduo](https://github.com/zhongduo)
 
 # Schedule
 
 | Week       | Contact                                               |
 | ---------- | ----------------------------------------------------- |
-| 2020-08-17 | [n3wscott](https://github.com/n3wscott)               |
-| 2020-08-24 | [vaikas](https://github.com/vaikas)                   |
-| 2020-08-31 | [grantr](https://github.com/grantr)                   |
-| 2020-09-07 | [lionelvillard](https://github.com/lionelvillard)     |
-| 2020-09-14 | [slinkydeveloper](https://github.com/slinkydeveloper) |
-| 2020-09-21 | [Harwayne](https://github.com/Harwayne)               |
-| 2020-09-28 | [evankanderson](https://github.com/evankanderson)     |
-| 2020-10-05 | [nachocano](https://github.com/nachocano)             |
-| 2020-10-12 | [matzew](https://github.com/matzew)                   |
-| 2020-11-16 | [lberk](https://github.com/lberk)                     |
-| 2020-11-23 | [pierDipi](https://github.com/pierDipi)               |
-| 2020-11-30 | [grantr](https://github.com/grantr)                   |
-| 2020-12-07 | [n3wscott](https://github.com/n3wscott)               |
-| 2020-12-14 | [vaikas](https://github.com/vaikas)                   |
-| 2020-12-21 | [aliok](https://github.com/aliok)                     |
-| 2020-12-28 | [aliok](https://github.com/aliok)                     |
+| 2021-01-04 | [n3wscott](https://github.com/n3wscott)               |
+| 2021-01-11 | [vaikas](https://github.com/vaikas)                   |
+| 2021-01-18 | [grantr](https://github.com/grantr)                   |
+| 2021-01-25 | [lionelvillard](https://github.com/lionelvillard)     |
+| 2021-02-01 | [slinkydeveloper](https://github.com/slinkydeveloper) |
```

Comment:
I might have some personal tasks during feb and I could be less available, can you push my turn to march?

Reply:
@slinkydeveloper maybe switch with @zhongduo ?

Reply:
Is March 15th ok? @slinkydeveloper

Reply:
sure